### PR TITLE
Kl 161596439 pm survey action buttonform

### PR DESCRIPTION
**Pull request by kahlouie**
## Description

Remove the pm survey form from the tsp app, and change the pm survey to use a wizard.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_populate_e2e
```

Use locator ENTPMS to fill out the PM survey from the tsp app with the button.

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* https://www.pivotaltracker.com/story/show/161596439

## Screenshots

enterpremovesurvey

### CODE REVIEW

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx [hunk 1]
```diff
@@ -125,12 +125,11 @@ class ShipmentInfo extends Component {
     super(props);
 
     this.assignTspServiceAgent = React.createRef();
-    this.enterPreMoveSurvey = React.createRef();
+    this.assignServiceMember = React.createRef();
```
Comment (kilbergr) [inline src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: What is this for? I think you can remove it .

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx [hunk 2]
```diff
@@ -14,7 +14,6 @@ import LoadingPlaceholder from 'shared/LoadingPlaceholder';
 import Alert from 'shared/Alert';
 import DocumentList from 'shared/DocumentViewer/DocumentList';
 import { withContext } from 'shared/AppContext';
-import PremoveSurvey from 'shared/PremoveSurvey';
```
Comment (kilbergr) [inline src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: Considering this is no longer shared, do you think it's worth moving into the mymove app folder for clarity's sake?

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx [hunk 3]
```diff
@@ -364,9 +353,12 @@ class ShipmentInfo extends Component {
                   </div>
                 )}
               {canEnterPreMoveSurvey && (
-                <button className="usa-button-primary" onClick={this.toggleEditPreMoveSurvey}>
-                  Enter pre-move survey
-                </button>
+                <FormButton
+                  FormComponent={PremoveSurveyForm}
+                  schema={this.props.shipmentSchema}
```
Comment (kilbergr) [inline src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: cool

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx (hunk 1 above)
Comment (kahlouie) [reply src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: hmmm, I thought it was  a merge conflict with your work! Glad you don't know what it is. will remove it.

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx (hunk 2 above)
Comment (kahlouie) [reply src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: good idea, I didn't even think about that!

**Comment by kahlouie** [comment]
@kilbergr changes made! :) Most of the work I've had for this project has been front end, and @sarboc and @Ronolibert have helped me tremendously :)

**Comment by kilbergr** [comment]
ONE LAST THING: 
Will you remove line 114 (`this.props.setEditPreMoveSurvey(false);`) from `Office/PremoveSurvey.jsx`? It's not used anymore.

**Pull request merged**